spec/controllers/reviews_controller_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+describe ReviewsController do
+  describe "POST create" do
+    context "with authenticated user" do
+      let(:video) { Fabricate(:video) }
+      let(:user) { Fabricate(:user) }
+      let(:log_in) { session[:user_id] = Fabricate(:user).id }
```

Comment:
You already have a user you can create on the line above. Why not use that? Then you can access the user directly in your specs.